src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -442,7 +486,7 @@ export function getSortField(
   }
 
   for (const alias in AGGREGATIONS) {
-    if (field.startsWith(alias)) {
+    if (field.startsWith(alias) && !field.startsWith('user_misery_prototype')) {
```

Comment:
user_misery aggregate function is not sortable

Reply:
Yes, this will make it sortable. The issue is this for loop will catch user_misery_prototype when aggregate `alias == 'user_misery'` (the legacy one) which is unsortable. Using the [number type field meta](https://github.com/getsentry/sentry/pull/24537/files#diff-69f7fa896f1221cf4acf5b839661f4fea2dd0da6b7ed7e25ca5eef010b99b8b9R494) to sort for now. When we expose user_misery_prototype to fields.tsx we can remove this check.

Reply:
To be more explicit in the short term, can we do something like

```
if (field.startsWith('user_misery_prototype')) {
  return true; // or what it should be here
}
```

This makes the intent here more explicit until it is exposed to discover.